docs/guides/options.md

```diff
@@ -1,138 +1,352 @@
 
+Determines whether or not the player has controls that the user can interact with. Without controls the only way to start the video playing is with the `autoplay` attribute or through the Player API.
 
-### loop ###
-The loop attribute causes the video to start over as soon as it ends. This could be used for a visual effect like clouds in the background.
-```html
-<video loop ...>
-or
-{ "loop": true }
+#### `height`
+
+> Type: `String|Number`
+
+Sets the display height of the video player in pixels.
+
+#### `loop`
+
+> Type: `Boolean`
+
+Causes the video to start over as soon as it ends.
+
+#### `muted`
+
+> Type: `Boolean`
+
+Will silence any audio by default.
+
+#### `poster`
+
+> Type: `String`
+
+A URL to an image that displays before the video begins playing. This is often a frame of the video or a custom title screen. As soon as the user hits "play" the image will go away.
+
+#### `preload`
+
+> Type: `String`
+
+Suggests to the browser whether or not the video data should begin downloading as soon as the `<video>` element is loaded. Supported values are:
+
+- `'auto'`
+
+  Start loading the video immediately (if the browser supports it). Some mobile devices will not preload the video in order to protect their users' bandwidth/data usage. This is why the value is called 'auto' and not something more conclusive like `'true'`.
+
+  _This tends to be the most common and recommended value as it allows the browser to choose the best behavior._
+
+- `'metadata'`
+
+  Load only the meta data of the video, which includes information like the duration and dimensions of the video.
+
+- `'none'`
+
+  Don't preload any data. The browser will wait until the user hits "play" to begin downloading.
+
+#### `src`
+
+> Type: `String`
+
+The source URL to a video source to embed.
+
+#### `width`
+
+> Type: `String|Number`
+
+Sets the display height of the video player in pixels.
+
+### Video.js-specific Options
+
+Each option is `undefined` by default unless otherwise specified.
+
+#### `aspectRatio`
+
+> Type: `String`
+
+Puts the player in [fluid](#fluid) mode and the value is used when calculating the dynamic size of the player. The value should represent a ratio - two numbers separated by a colon (e.g. `"16:9"` or `"4:3"`).
+
+#### `children`
+
+> Type: `Array`
+
+#### `fluid`
+
+> Type: `Boolean`
+
+When `true`, the Video.js player will have a fluid size. In other words, it will scale to fit its container.
+
+#### `inactivityTimeout`
+
+> Type: `Number`
+
+Video.js indicates that the user is interacting with the player by way of the `"vjs-user-active"` and `"vjs-user-inactive"` classes and the `"useractive"` event.
+
+The `inactivityTimeout` determines how many milliseconds of inactivity is required before declaring the user inactive.
+
+#### `language`
+
+> Type: `String`
+
+A [two-letter code][lang-codes] matching one of the available languages in the player. This sets the initial language for a player, but it can always be changed.
```

Comment:
Not necessarily two-letter. And while it's true the player's language can be changed later on, that can lead to confusion as localisable text does not get updated.